lib/rabbitmq/cli/command_behaviour.ex

```diff
@@ -29,7 +29,9 @@ defmodule RabbitMQ.CLI.CommandBehaviour do
                       scopes: 0,
                       usage_additional: 0,
                       switches: 0,
-                      aliases: 0
+                      aliases: 0,
+                      required_rabbit_app_state: 2,
+                      validate_execution_environment: 2
```

Comment:
> It's not something optional

`rabbitmq-diagnostics memory_breakdown` doesn't care about the `rabbit` app at all, and does not have an `offline` switch, for one example. I think several of those diagnostics commands don't care about the `rabbit` app.